app/src/main/java/net/aiscope/gdd_app/ui/mask/customview/MaskCustomView.kt

```diff
@@ -32,7 +36,7 @@ class MaskCustomView @JvmOverloads constructor(
     var onMaskingActionFinishedListener: OnTouchListener? = null
     private val maskLayer = MaskLayer(context, imageMatrix)
     private var currentMode: Mode = Mode.Draw
-    private lateinit var drawableDimensions: Pair<Int, Int>
+    private lateinit var drawableDimensions: Size
```

Comment:
Do you mind renaming this to drawableSize, please?

Reply:
Good catch! :) Fixed.

Reply:
Great! Thanks!!